.github/workflows/verible-ci.yml

```diff
@@ -12,15 +12,17 @@ on:
   workflow_dispatch:
 
 env:
-  DOCKER_BUILDKIT: 1
   BOT_USER: "Deployment Bot"
   BOT_EMAIL: "verible-dev@googlegroups.com"
+  GHA_MACHINE_TYPE: "n2-highcpu-8"
 
 jobs:
 
 
   VerifyFormatting:
-    runs-on: ubuntu-20.04
+    #runs-on: ubuntu-20.04
```

Comment:
is this commented-out line a debugging leftover ?

Reply:
This is a leftover indeed. I will remove it shortly.